enterprise/server/remote_execution/runner/runner.go

```diff
@@ -69,6 +69,9 @@ var (
 	// can't be added to the pool and must be cleaned up instead.
 	maxRunnerMemoryUsageBytes = flag.Int64("executor.runner_pool.max_runner_memory_usage_bytes", 0, "Maximum memory usage for a recycled runner; runners exceeding this threshold are not recycled.")
 	podmanWarmupDefaultImages = flag.Bool("executor.podman.warmup_default_images", true, "Whether to warmup the default podman images or not.")
+
+	overlayfsEnabled  = flag.Bool("executor.workspace.overlayfs_enabled", false, "Enable overlayfs for executor workspaces. This provides a stronger guarantee that the executor's local cache cannot be modified by actions. Only available on Linux, and requires the executor to have mount() permissions as well as the overlayfs kernel module to be enabled.")
+	overlayfsAnonOnly = flag.Bool("executor.workspace.overlayfs_anonymous_only", false, "Restrict usage of overlayfs to unauthenticated requests only.")
```

Comment:
I'm not sure I understand why we need this flag? It seems like we want:

- a flag to flip this on/off for anon
- a platform property so anyone can turn this on (if it's enabled)

Reply:
can we do this in a followup? I think we should test this first, on our own stuff, on anon only, to begin with. (I'm fine if you want to add this functionality later in a followup)

Reply:
sg - kept the anon-only flag and removed the executor-wide one.